### PR TITLE
Add integration tests for NPM E2E flow

### PR DESCRIPTION
**Pull request by rootAvish**
Signed-off-by: Avishkar Gupta <avgupta@redhat.com>

### CODE REVIEW

**Comment by rootAvish** [comment]
[test]

File: integration-tests/features/steps/stack_analysis.py [hunk 1]
```diff
@@ -90,6 +90,7 @@ def perform_valid_manifest_post(context, manifest, url):
 def send_manifest_to_stack_analysis(context, manifest, name, endpoint, use_token):
     """Send the selected manifest file to stack analysis."""
     filename = 'data/{manifest}'.format(manifest=manifest)
+    print("filename is: {}".format(filename))
```
Comment (tisnik) [inline integration-tests/features/steps/stack_analysis.py]: leftover?

**Comment by rootAvish** [comment]
@tisnik Thanks for the review, done but looks like somebody broke the CICO duffy code itself 🙁 
```
++ cico node get -f value -c ip_address -c comment
string indices must be integers
```

I'll try again in some time.

**Comment by rootAvish** [comment]
[test]

**Comment by rootAvish** [comment]
@tisnik Looks like it passed, please see again.

**Comment by tisnik** [comment]
Thanks!

**Pull request merged**